homeassistant/components/firmata/__init__.py

```diff
@@ -159,10 +197,18 @@ async def handle_shutdown(event) -> None:
         hass.async_create_task(
             hass.config_entries.async_forward_entry_setup(config_entry, "binary_sensor")
         )
+    if CONF_SENSORS in config_entry.data:
```

Comment:
Can we use a for loop that loops over the platform config keys?

Reply:
The key in `config_entry.data` is plural, but the values for `async_forward_entry_setup` are singular, so I'm not sure how many lines it would save after creating a dictionary to loop over. I can if you'd like.

Reply:
We can break out the dictionary to a constant so we'll save some lines locally in this function.

Reply:
Ah, I didn't think about doing it that way.